use-context/Exercise1/problem/src/providers/userProvider/UserProvider.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { createContext, useEffect, useState } from "react";
+
+const loginUrl = "https://cowbird-staging.fly.dev/groups/bitovi/mocks/getUser";
+const usersUrl = "https://cowbird-staging.fly.dev/groups/bitovi/mocks/getUsers";
```

Comment:
It’d be good to get some info about this dependency documented in the readme, e.g. what is this dependency / where is the code / etc.?